src/dns/test/dns_bind_test.cc

```diff
@@ -231,8 +231,9 @@ TEST_F(DnsBindTest, Config) {
 }
 #endif
 
+#ifndef _WIN32
 // TODO(WINDOWS): JW-1168: NamedConfig is not used in the Agent
```

Comment:
You kept this comment...

Reply:
I ifndefed two methods, but I should move this comment one line above